src/redux/covidData/covidData.js

```diff
@@ -0,0 +1,47 @@
+const apiEndpoint = 'https://api.covid19tracking.narrativa.com/api/';
+const initialState = { countries: [], totalConfirmed: 0 };
+
+export const FETCH_DATA = 'data/FETCH_DATA';
+const ADD_COUNTRY = 'countries/ADD_REGION';
+const SET_TOTAL = 'numbers/SET_TOTAL_CONFIRMED';
+
+const fetchCovidData = async () => {
+  const today = new Date().toISOString().split('T')[0];
+  let res = await fetch(`${apiEndpoint}/${today}`);
+  res = await res.json();
+  return res;
+};
+
+export const addCountry = (country) => ({
+  type: ADD_COUNTRY,
+  payload: country,
+});
+
+const setTotal = (total) => ({
+  type: SET_TOTAL,
+  payload: total,
+});
```

Comment:
[OPTIONAL]

- It would be better if you separate the reduction action and API call. It's always better to do the API call in a separate file and use it by importing the function in your react component.

Reply:
Done